phpcs.xml

```diff
@@ -6,4 +6,8 @@
 
     <rule ref="PHPCompatibility"/>
     <rule ref="PSR2"/>
+
+    <rule ref="PSR2">
+      <exclude name="Generic.Files.LineLength"/>
```

Comment:
I've opted to ignore long lines as it doesn't look like it's really been enforced previously.